src/pages/workspaces/workspace/workflows/LaunchAnalysisModal.js

```diff
@@ -114,8 +125,14 @@ const LaunchAnalysisModal = ({
           p(['Note that metadata about this run will be stored in the US.'])
         ])]) : 'Loading...'
     ]),
-    div({ style: { margin: '1rem 0' } }, [
-      'This will launch ', b([entityCount]), ` analys${entityCount === 1 ? 'is' : 'es'}`,
+    div({ style: { margin: '1rem 0', ...maybeWarningStyle } }, [
+      warnDuplicateAnalyses && icon('warning-standard', { size: 16, style: { color: colors.warning(), marginRight: '0.5rem' } }),
+      warnDuplicateAnalyses && 'Warning! ',
+      'This will launch ',
+      b([entityCount]),
+      warnDuplicateAnalyses && span({ style: { fontWeight: 'bold', textDecoration: 'underline' } }, [' duplicate']),
+      entityCount === 1 ? ' analysis' : ' analyses',
+      warnDuplicateAnalyses && ' of the same set of constant inputs',
```

Comment:
nit: Rather than doing word-by-word conditions, It might be cleaner to just have two separate branches, even if there's a little duplication in the content.